src/RestaurantButton.jsx

```diff
@@ -0,0 +1,5 @@
+export const RestaurantButton = ({ onClick }) => (
```

Comment:
I noticed we export our components differently across files, this isn't a big problem, but consistency is a good thing in code, it keeps clean and easy to understand 😊

Reply:
Sure I will do that 😊